src/scripts/surf.ts

```diff
@@ -0,0 +1,74 @@
+// Description:
+//  Surf server stat comparison
+//
+// Commands:
+//  hubot surf me - compare surfers
+//
+// Author:
+//  Steve Shipsey
+
+import Axios, { AxiosResponse } from 'axios';
+import * as Cheerio from 'cheerio';
+import { Response, Robot } from 'hubot';
+import { table, TableUserConfig } from 'table';
+
+const baseUrl = 'https://snksrv.com/surfstats';
+const surfers = [
+  'STEAM_1:0:6030042',
+  'STEAM_1:0:1166568',
+];
+const headers = ['Surfer', 'Points', 'Map Completions'];
+
+const fetchSurfer = async (steamId: string) => await Axios.get<any>(`${baseUrl}`, {
+  params: {
+    id: steamId,
```

Comment:
if you name the parameter `id` then you can just do `params: { id, view: 'profile' }`

Reply:
ooh, TIL. or maybe today i remembered.